### ISSUE TITLE
Namalsk server  keyed Mozzie leaks fuel with no option to repair anything (all green), don't know if this existed before as always packed before much fuel consumed. Did not seem to exist before Vhic was filled by gas pump either. could that somehow accelerate/inflict damage? As stands it would completely drain fuel in 10-20 mins.

### ISSUE THREAD

**Issue by hamdammer** [BetterDeadThanZed/serverissues#3]

**Comment by BetterDeadThanZed** [comment]
The deployed vehicles will eventually not be able to be keyed anyway. They were never intended to be able to be keyed. I am waiting to test a fix once I've got the test server working, then all deployed mozzies will revert to unlocked. As far as fixing them, they should be repairable just like a spawned version so if it doesn't  work, then it's a bug in the game.

**Comment by ekroemer** [comment]
I've had a look into the DB and confirmed my suspicion: the mozzie has a damage > 0 but the hitpoints field is emtpy ([]).

I suspect the following:
  - in Arma vehicles can get damaged (duh!)
  - this is locational damage (blown tire,...)
  - DayZ/Epoch can repair locational damage
  - but only for parts that are explicitely handled in the mod's code (entries in hitpoints DB field)
  - parts that are in Arma but not handled by DayZ repair stay damaged, no matter what you do
  - and thus cause fuel loss

Benign example: MV22
  - has only 1 repairable part (scrap)
  - can be visibly damaged (trails smoke, has bullet riddled skin)
  - but Epoch repair does not give an option to use scrap to repair
  - however, once you touch the repair-button, the plane magically goes to full health

Malign example: any tank
  - when you have your tank shot at, the 'gun' may be damaged, making it droop and unusable
  - you can repair all Epoch offers you fully,
  - but the gun stays dead.

The only in-game way I know to fully repair a vehicle that is such afflicted, is the repair station script. Apparently this addresses all kinds of damage a vehicle can take, e.g. the gun will work again.

One can, and I have done this for the mozzie, set the damage field in the DB to 0.0 and all will be well after the next restart (as long as you didn't move the thing after I changed the DB ;-)

tldr;
this is Arma, known from other circumstances, has nothing to do with the way the mozzie came into in existence and can't be fixed (other than by removing all vehicles that have parts not covered by DayZ or installing/using the repair station script).

**Comment by hamdammer** [comment]
I just left it where it could be fueled by pump/generator, then checked it today and it fills full and seems to hold fuel as new....figured it was only a temp situation one way or another, but thought the info might help direct a good solution, thanks for the attention and remedies/changes applied.

**Comment by BetterDeadThanZed** [comment]
Keys can no longer be made for mozzies as of this morning, but you can pack mozzies again, so that means that if you pack it, the damage will be repaired and fuel level will be refilled. Since it's not being written to the database, there's no record of damage once it's packed. This won't apply to mozzies that exist already, which have keys made for them.
